scripts/config/metrictank-package.ini

```diff
@@ -356,6 +358,8 @@ password = cassandra
 create-keyspace = true
 # File containing the needed schemas in case database needs initializing
 schema-file = /etc/metrictank/schema-idx-cassandra.toml
+# instruct the driver to not attempt to get host info from the system.peers table
+disable-initial-host-lookup = false
```

Comment:
i'm not a big fan of double negatives. can we call this `initial-host-lookup = true` ?

Reply:
why do we need this setting anyway?

Reply:
the config setting used by gocql is "DisableInitialHostLookup" so i think it makes sense to also use that.